Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM elixir:1.9-alpine as builder
+
+ENV MIX_ENV prod
+
+RUN apk add --no-cache build-base curl
+RUN mix local.hex --force
+RUN mix local.rebar --force
+RUN mkdir -p /mini_repo
+
+WORKDIR /mini_repo
+
+ADD config  ./config/
+ADD lib  ./lib/
+ADD priv ./priv/
+ADD rel ./rel/
+COPY mix.exs mix.lock ./
+
+RUN mix deps.get
+RUN mix compile
+RUN mix release
+
+FROM alpine:3.9
+
+RUN apk add --no-cache ncurses-libs openssl
+
+COPY --from=builder /mini_repo/_build/prod/rel/mini_repo /app/
```

Comment:
The running container should use a non-root user. That can be done with something like:

``` dockerfile
RUN addgroup -g 1000 mini_repo \
 && adduser -D -H -u 1000 -G mini_repo mini_repo
USER 1000
COPY --chown=1000:1000 --from=builder /mini_repo/_build/prod/rel/mini_repo /app/
```